src/components/config/parse.ts

```diff
@@ -24,14 +24,82 @@
 
 import { Config } from '../../interfaces/config'
 import { readFileSync } from 'fs'
+import { Probe } from '../../interfaces/probe'
+import { DEFAULT_THRESHOLD } from '../../looper'
 
-export const parseConfig = (configPath: string): Config => {
+const defaultNotification = [
+  {
+    type: 'desktop',
+    id: 'unique-id-desktop-monika-notif',
+  },
+]
+
+const getConvertedProbeFromPostmanItem = (item: any) => {
+  const req = item.request
+  const probe: Probe = {
+    id: item.name,
+    name: item.name,
+    requests: [
+      {
+        url: req.url.raw,
+        method: req.method,
+        headers: req?.header?.reduce(
+          (obj: any, it: any) => Object.assign(obj, { [it.key]: it.value }),
+          {}
+        ),
+        body: req?.body?.raw || JSON.parse('{}'),
+        timeout: 10000,
+      },
+    ],
+    incidentThreshold: DEFAULT_THRESHOLD,
+    recoveryThreshold: DEFAULT_THRESHOLD,
+    alerts: ['status-not-2xx', 'response-time-greater-than-285-ms'],
+  }
+
+  return probe
+}
+
+export const parseConfigFromPostman = (configString: string): Config => {
+  try {
+    const parsed = JSON.parse(configString)
+    const probes: Probe[] = []
+
+    parsed.item.forEach((item: any) => {
+      if (item.item) {
```

Comment:
I think we need a recursive function here because the depth of grouping can be arbitrary, it can be more than 2 level deep

```json
"item": [
		{
			"item": [
				{
					"item": [
						{
							"item": [
				        {
                 "item": [
				           {

```

Reply:
I haven't thought of that, will look into that. Nice catch

Reply:
@armedi it's fixed. please recheck